code/modules/projectiles/medbeam.dm

```diff
@@ -2,6 +2,7 @@
 	name = "Medical Beamgun"
 	desc = "Delivers medical nanites in a focused beam."
 	icon = 'icons/obj/chronos.dmi'
+	origin_tech = "combat=6;materials=7;powerstorage=8;programming=7;biotech6"//The ultimate within medical design
```

Comment:
Yeah, no.

This allows you to hit materials 8 and programming 8; these should all be 1 tech level lower than they are---there's intentional inconsistencies beteween origin tech and tech required to actually produce the item.